livekit_agent.proto

```diff
@@ -74,25 +78,49 @@ enum JobStatus {
     JS_FAILED = 2;
 }
 
+message SimulateJobRequest {
+    JobType type = 1;
+    string room_id = 2;
+    string participant_id = 3;
+}
+
+message PingRequest {
+    double current_load = 1;
+}
+message PingResponse {}
+
 message RegisterWorkerRequest {
     JobType type = 1;
     string worker_id = 2;
     string version = 3;
     string name = 4;
+    uint32 ping_interval = 5;
+    optional string namespace = 6;
```

Comment:
should the worker indicate if they support migrations or not? thinking that if a job cannot be migrated (because they are not ready to handle the state), then it would be unexpected to receive another request for it.

Reply:
There is already the should_retry value when accepting a specific job. Mb this should be renamed?

Reply:
ah ha.. ok. maybe `supports_resume`? (signals if the current job can be retried..) having it on the Job is even more flexible than doing it on worker.

thinking `resume` because you have `is_resume` on the job request itself.

Reply:
renamed to supports_resume